README.rst

```diff
@@ -89,15 +89,15 @@ The following Python modules are required:
 Optional requirements include:
 
 * azure_ for Microsoft Azure object storage
-* boto_ for AWS S3 (or Ceph-S3) object storage
+* botocore_ for AWS S3 (or Ceph-S3) object storage
```

Comment:
Please note this change in NEWS as well.

Reply:
Updated NEWS as well